<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 220, 243);
colorr: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_c">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(234, 163, 75);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pm">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pnt">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color: rgb(255, 220, 243);
color: rgb(40, 90, 78)

}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bs">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(234, 163, 75);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n2">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>101</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n3">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>101</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_r">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>101</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>101</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n5">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>142</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n6">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>142</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_d">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>142</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>142</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n8">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>183</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n9">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>183</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>183</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>183</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>224</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n0">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>224</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 192, 195);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_p">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>224</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>224</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>265</y>
      <width>142</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(234, 163, 75);
color: rgb(40, 90, 78);
MainWindow::on_pushButton_c_clicked()</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_square">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>265</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reverse">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>265</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>306</y>
      <width>284</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1 px solid grey;
background-color: rgb(159, 166, 227);
color: rgb(40, 90, 78);
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
